src/web/client/extension.ts

```diff
@@ -66,13 +64,16 @@ export function activate(context: vscode.ExtensionContext): void {
                                 context.globalState.update(IS_FIRST_RUN_EXPERIENCE, false);
                                 WebExtensionContext.telemetry.sendInfoTelemetry("StartCommand", { 'commandId': 'workbench.action.openWalkthrough', 'walkthroughId': 'microsoft-IsvExpTools.powerplatform-vscode#PowerPage-gettingStarted' });
                             }
-
                             await vscode.window.withProgress({
                                 location: vscode.ProgressLocation.Notification,
                                 cancellable: true,
                                 title: localize("microsoft-powerapps-portals.webExtension.fetch.file.message", "Fetching your file ...")
                             }, async () => {
                                 await vscode.workspace.fs.readDirectory(WebExtensionContext.rootDirectory);
+                                await NPSService.setEligibility();
```

Comment:
can we keep it outside the withProgress notification flow for fetching the file-  we dont need to show the notification while we are preparing the survey prompt